pkg/apis/uiplugin/v1alpha1/types.go

```diff
@@ -132,6 +135,43 @@ type LokiStackReference struct {
 	Name string `json:"name"`
 }
 
+// MonitoringConfig contains options for configuring the monitoring console plugin.
+type MonitoringConfig struct {
+	// Alertmanager points to the alertmanager instance of which it should create a proxy to.
+	//
+	// +kubebuilder:validation:Required
```

Comment:
This would be required only on the case the monitoring plugin has been started with the acm-alerting flag. Should we consider doing this optional or add some validation based on the features enabled?